### ISSUE TITLE
Installation fails on Win10 & julia 1.4.2

### ISSUE THREAD

**Issue by Kelvyn88** [SciML/DifferentialEquations.jl#635]
![WhatsApp Image 2020-07-13 at 7 05 27 PM](https://user-images.githubusercontent.com/11316335/87365555-2785a100-c53c-11ea-9dc6-7d424854a9a9.jpeg)

**Comment by ChrisRackauckas** [comment]
@chriselrod looks like one for you again.

FWIW, I develop on Win10 + Julia 1.4.2, so the title is incorrect at least. I know it works there. It's probably something more related to the CPU then? What's your `versioninfo()`?

**Comment by chriselrod** [comment]
Probably more related to how Julia was installed?

The error seems to have been in loading `LLVM.dll`.

Out of curiosity, can you `using LLVM`?

**Comment by Kelvyn88** [comment]
Hi,

The problem was presented in 2 new computers running Win10. Fresh install of julia 1.4.2. 

Let me check.

**Comment by chriselrod** [comment]
Assuming `using LLVM` fails, mind filing an issue there?

**Comment by chriselrod** [comment]
I dropped the LLVM dependency from VectorizationBase.
Please confirm things work after updating your packages.

**Comment by tomjcamp** [comment]
I am having a similar issue installing this in the macos version with Julia 1.4.2. I have just downloaded this from julialang.org, 
and have not been able to find a fix for this on my end. I did notice that the author for Pretty.jl seems to have implemented a change from pprint_impl to pp_impl, for what its worth. 

![image](https://user-images.githubusercontent.com/40920347/88883986-40a87600-d1fb-11ea-86cb-3615b6cabff2.png)
